lib/pure/os.nim

```diff
@@ -3194,6 +3208,30 @@ proc setLastModificationTime*(file: string, t: times.Time) {.noNimScript.} =
     discard h.closeHandle
     if res == 0'i32: raiseOSError(osLastError(), file)
 
+func isValidFilename*(path: string, maxLen = 259.Positive): bool {.since: (1, 1).} =
```

Comment:
`path` implies things like "/foo/bar.txt" to me, but that isn't what this function tests. You should rename this to `filename`.

Reply:
I was thinking the same, great suggestion, will rename.